packages/appeals-service-api/src/models/entities/lpa-entity.js

```diff
@@ -1,66 +1,77 @@
 class LpaEntity {
-    #_id;
-    #objectId;
-    #lpa19CD;
-    #lpaCode;
-    #name;
-    #email;
-    #domain;
-    #inTrial;
+	#_id;
+	#objectId;
+	#lpa19CD;
+	#lpaCode;
+	#name;
+	#email;
+	#domain;
+	#inTrial;
 
-    constructor(_id, objectId, lpa19CD, lpaCode, name, email, domain, inTrial) {
-        this.#_id = _id;
-        this.#objectId = objectId;
-        this.#lpa19CD = lpa19CD;
-        this.#lpaCode = lpaCode;
-        this.#name = name;
-        this.#email = email;
-        this.#domain = domain;
-        this.#inTrial = inTrial;
-    }
+	constructor(_id, objectId, lpa19CD, lpaCode, name, email, domain, inTrial) {
+		this.#_id = _id;
+		this.#objectId = objectId;
+		this.#lpa19CD = lpa19CD;
+		this.#lpaCode = lpaCode;
+		this.#name = name;
+		this.#email = email;
+		this.#domain = domain;
+		this.#inTrial = inTrial;
+	}
 
-    static createFromJson(json) {
-        return new LpaEntity(
-            json._id,
-            json._objectId,
-            json.lpa19CD,
-            json.lpaCode,
-            json.name,
-            json.email,
-            json.domain,
-            json.inTrial
-        );
-    }
+	static createFromJson(json) {
+		return new LpaEntity(
+			json._id,
+			json._objectId,
+			json.lpa19CD,
+			json.lpaCode,
+			json.name,
+			json.email,
+			json.domain,
+			json.inTrial
+		);
+	}
 
-    getLpa19CD() { return this.#lpa19CD; }
-    getLpaCode() { return this.#lpaCode; }
-    getName() { return this.#name; }
-    getEmail() { return this.#email; }
-    getDomain() { return this.#domain; }
+	getLpa19CD() {
+		return this.#lpa19CD;
+	}
+	getLpaCode() {
+		return this.#lpaCode;
+	}
+	getName() {
+		return this.#name;
+	}
+	getEmail() {
+		return this.#email;
+	}
+	getDomain() {
+		return this.#domain;
+	}
 
-    isInTrial() { return this.#inTrial; }
-    getCountry() { 
-        if (this.#lpa19CD.startsWith('E')) {
-            return "England"
-        } else if (this.#lpa19CD.startsWith('W')) {
-            return "Wales"
-        } else {
-            return ""
-        }
-    }
+	isInTrial() {
+		return this.#inTrial;
+	}
+	getCountry() {
+		const isWales = new RegExp('W[0-9]{8}');
```

Comment:
Probably not an issue but this regex would check for any string which contains a W followed by 8 numbers, for example, `xxxxW1234567890`. It could be made more specific by defining the start and end of the string like this:
`const isWales = new RegExp('^W[0-9]{8}$');`